lib/ip.go

```diff
@@ -193,25 +201,27 @@ IP:
 // RandIP6Range returns a list of randomly generated IPv6 addresses within
 // the range specified by `startIP` and `endIP`.
 //
-// note: `EvalIP6` must be called before this function as this function assumes
-// `startIP` and `endIP` is a correct range.
-func RandIP6Range(ipRange IP6RangeInt, noBogon bool) (net.IP, error) {
+// note: `NewIP6RangeInt` must be called before this function as this function
+// assumes `startIP` and `endIP` is a correct range.
+func RandIP6Range(ipRange IP6RangeInt, noBogon bool) net.IP {
 	randIP := new(big.Int)
 	randIP.SetBytes(RandIP6(noBogon))
 	tmp := new(big.Int)
 	tmp.Sub(ipRange.endIP, ipRange.startIP)
-	if tmp.Cmp(big.NewInt(0)) <= 0 {
+	if tmp.Cmp(big.NewInt(0)) == 0 {
+
+		// convert multi-precision byte form into 16-byte IPv6 form.
```

Comment:
I meant to put this comment below, where we initialize the `randIPBytes` in the final return, not inside this `if` clause. However, it wouldn't hurt to have both. But then you should erase the newline above this comment.